guides/source/configuring.md

```diff
@@ -952,6 +952,16 @@ text/javascript image/svg+xml application/postscript application/x-shockwave-fla
 
 * `config.active_storage.draw_routes` can be used to toggle Active Storage route generation. The default is `true`.
 
+* `config.active_storage.resolve_name` can be used to globally change how Active Storage files are delivered.
+
+  ```ruby
+  config.active_storage.resolve_name = :rails_storage_proxy
+  ```
+
+  The default is `nil`. In practive will behave similar to setting `:rails_storage_redirect` Can include any of these options:
```

Comment:
How about we just set the default to `rails_storage_redirect` in the config, then you don't need to guard for nil.